pkg/services/dashboards/database/database_test.go

```diff
@@ -1065,11 +1063,10 @@ func insertTestDashboard(t *testing.T, dashboardStore dashboards.Store, title st
 }
 
 func insertTestDashboardForPlugin(t *testing.T, dashboardStore dashboards.Store, title string, orgId int64,
-	folderId int64, isFolder bool, pluginId string) *dashboards.Dashboard {
```

Comment:
I think you need to introduce a `folderUID` instead or keep both

Reply:
I should, I as considering it, but it still passed without it